Examples/Java/Sources/Pin.java

```diff
@@ -957,7 +953,7 @@ public Pin read(@NonNull JsonReader reader) throws IOException {
         }
     }
 
-    public static final class PinAttributionObjects<R> {
```

Comment:
`<R>` isn't really necessary here. We can adjust the signature of `matchPinAttributionObjects` instead.